src/olympia/addons/models.py

```diff
@@ -1704,10 +1704,14 @@ def git_extraction_is_in_progress(self):
 
 
 @receiver(signals.version_changed, dispatch_uid='version_changed')
-def version_changed(sender, **kw):
+def version_changed(sender, instance, **kw):
```

Comment:
possibly missing something, but why are we changing `sender` to be the Addon class and then passing the instance separately?  Is this just clean-up because `sender` is typically the class?

Reply:
Yes - consistency because I made the mistake thinking sender was a class initially when testing when it wasn't at the time :)